repo/fsrepo.go

```diff
@@ -336,7 +340,7 @@ func (r *FSRepo) loadVersion() (uint, error) {
 
 	version, err := strconv.Atoi(strings.Trim(string(file), "\n"))
 	if err != nil {
-		return 0, err
+		return 0, errors.New("corrupt version file: version is not an integer")
```

Comment:
Do we want to `Wrap` this error instead? Might be wise unless we are sure `not an integer` is the only type of error we can get back from `Atoi`. 

The assertion in the test below can still be made using `Contains`.

Reply:
Good point.  It's not exactly the only type you can get back from [Atoi](https://golang.org/src/strconv/atoi.go?s=5007:5039#L192).  You can also get a range error if the integer is too big.

I am plowing ahead with this because filecoin will not have a repo version bigger than maxint in my lifetime and being slightly less precise is worth the cost of hiding the internally messy `strconv.Atoi: parsing "xxx": invalid syntax` error message from the user.